tcms/rpc/tests/test_testexecution.py

```diff
@@ -36,6 +36,26 @@ def test_add_comment_with_pk_as_int(self):
         self.assertEqual("Hello World!", first_comment.comment)
 
 
+class TestExecutionGetComments(APITestCase):
+    """ Test TestExecution.get_comments """
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        self.comments = ["Text for first comment", "Text for second comment"]
+
+        self.execution = TestExecutionFactory()
+        for comment in self.comments:
+            self.rpc_client.TestExecution.add_comment(self.execution.pk, comment)
+
+    def test_get_comment(self):
+        comments = self.rpc_client.TestExecution.get_comments(self.execution.pk)
+
+        self.assertEqual(len(self.comments), len(comments))
+        for comment in comments:
+            self.assertTrue(comment['comment'] in self.comments)
```

Comment:
Also needs a scenario covering the ObjectNotFound case when the method argument doesn't match an existing TestExecution.